src/github_review_prompts/core/prompt_engine.py

```
@@ -700,6 +744,577 @@ def generate_main_prompt(
+## 📋 統合対応指針
+
+### 🔒 セキュリティファースト原則
+1. **範囲外コメント優先**: プラットフォーム制限により見落としやすいため、最優先で対応
+2. **段階的対応**: 🔴緊急 → 🟡重要 → 🟢低優先の順で処理
+3. **影響範囲検証**: 各修正が他の箇所に与える影響を慎重に確認
+
+### 📊 対応完了の報告形式
+各範囲外コメントの対応完了時は以下の形式で報告してください：
+
+```
+✅ 範囲外TODO #{todo_number}: {title}
```

Ruff (0.12.2)

Undefined name `todo_number`

(F821)